package.json

```diff
@@ -58,14 +58,14 @@
     "@types/graphql-upload": "^8.0.6",
     "@types/jest": "^26.0.24",
     "@types/node": "^16.4.3",
-    "apollo-server-express": "^3.0.2",
+    "apollo-server-express": "^3.5.0",
     "body-parser": "^1.19.0",
     "doctoc": "^2.0.1",
     "dripip": "^0.10.0",
     "express": "^4.17.1",
     "fetch-cookie": "^1.0.0",
     "get-port": "^5.1.1",
-    "graphql": "^15.5.1",
+    "graphql": "16",
```

Comment:
This way of defining a dependency range is not described in the npm documentation (https://docs.npmjs.com/cli/v8/configuring-npm/package-json#dependencies). Should this be `^16.0.0` instead?